servlet/src/main/scala/org/http4s/servlet/Http4sServlet.scala

```diff
@@ -1,21 +1,19 @@
 package org.http4s.servlet
 
+import _root_.io.chrisdavenport.vault._
```

Comment:
Mostly useless comment: Do we want to just sort our _root_ imports at the top?  I usually put those inline without the _root_, but yours is even simpler to automate.  I suspect that one's not necessary without an `fs2._`.

Reply:
`_root_` is only necessary if the import is below `import fs2._` which imports the `io` scope, or `import specs2._` which also has an `io` scope. 

Otherwise, it can be imported normally. Both circe and my packages have this unpleasant arrangement due to `io` domain, and `io` as a concept.